wikitemplate.md

```diff
@@ -22,21 +22,21 @@
 
 ### Extensions/Plugins
 
-- [ ] Verify pdfium, Torrent viewer extensions are installed automatically on fresh profile and cannot be disabled
+- [ ] Verify pdfium, Torrent viewer extensions are installed automatically on fresh profile and cannot be disabled (they don't show up in `brave://extensions`)
 - [ ] Verify older version of an extension gets updated to new version via Google server
-- [ ] Verify older version of an extension gets updated to new version via Brave server 
-- [ ] Verify that `magnet` links and `.torrent` files correctly opens WebTorrent and you're able to download the file(s)
-  - **Tip:** Free torrents available via https://webtorrent.io/torrents/
+- [ ] Verify older version of an extension gets updated to new version via Brave server
+- [ ] Verify that `magnet` links and `.torrent` files correctly open WebTorrent and you're able to download the file(s)
+  - **Tip:** Free torrents available via https://torrent.ubuntu.com/
```

Comment:
Not sure when the link was updated but the correct link is `https://webtorrent.io/free-torrents` not `https://webtorrent.io/torrents`

Reply:
Thank you!  Fixed in https://github.com/brave/qa-resources/pull/203/commits/456b4116abee4dbdc33bdaddb33aedbd6b69ad06.

(When we merge this, we can quash the 4 commits into 1 via GitHub, hopefully, right?  I'm not great at rebasing via command-line.)

Reply:
No squash and merge is not enabled on the repo. Has to be done via command line. Or just merge it all at once